packages/common/package.json

```diff
@@ -19,6 +19,7 @@
     "graphql": "^15.5.1",
     "graphql-request": "^3.4.0",
     "history": "^5.0.0",
+    "kbar": "^0.1.0-beta.5",
```

Comment:
that was good timing!

Reply:
I made an issue and it was sorted within a few hours 🤓  😁